onnx/onnx-data.in.proto

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) ONNX Project Contributors.
+// Licensed under the MIT license.
+
+syntax = "proto2";
+
+package {PACKAGE_NAME};
+// #if ONNX-ML
+import "onnx/onnx-ml.proto";
+// #else
+import "onnx/onnx.proto";
+// #endif
+
+// This file contains the proto definitions for MapProto and
+// SequenceProto. These protos are used to represent the data structures
+// of maps and sequence for use in test data or ModelProto.
+
+// Sequences
+//
+// Defines a dense, ordered, collection of elements that are of homogeneous types.
+// Sequences can be made out of tensors, maps, or sequences.
```

Comment:
so if it's "TENSOR", the tensor's element type should also be the same, but can be different shapes, right? if yes, I'd suggest to add this clarification in the comments too.

Reply:
The clarification can point to https://github.com/onnx/onnx/blob/master/docs/IR.md#static-tensor-shapes for a more complete description. Whether the tensors can have different shapes or not depends on the type/shape associated with the corresponding "ValueInfo".

Reply:
For example, "Sequence<Tensor<float, [M,N]>" means that all tensors have same shape. But "Sequence<Tensor<float, [*,*]>" means they can have different shapes (but all of rank 2), where the "*" means the corresponding dimension has no symbolic/constant value.

Reply:
Finally, "Sequence<Tensor<float, omitted>>" means that the different tensors can have different ranks, when the "shape" itself is omitted from the tensor-type.

Reply:
Addressed and included these points in the recent update.